haystack/utils/docker.py

```diff
@@ -0,0 +1,15 @@
+
+def cache_models():
+    """
+    Small function that caches models and other data.
+    Used only in the Dockerfile to include these caches in the images.
+    """
+    # download punkt tokenizer
+    import nltk
+    nltk.download('punkt', download_dir='/usr/nltk_data')
```

Comment:
I am seeing some surprising messages in my log file when running a container (both with your PR or on master branch). It tries to download nltk tokenizers punkt although it should not:
```
[nltk_data] Downloading package punkt to /root/nltk_data...

[nltk_data]     Unzipping tokenizers/punkt.zip.
```
I understand that your code did not really change that behavior compared to master but maybe you could still quickly check whether the code tries to download nltk despite we want it to be cached? I could imagine it's because of the user and the different path names `'/usr/nltk_data'` and `'/root/nltk_data'`